Extensions/Leaderboards/leaderboardstools.ts

```diff
@@ -0,0 +1,75 @@
+namespace gdjs {
+  const logger = new gdjs.Logger('Leaderboards');
+  export namespace evtTools {
+    export namespace leaderboards {
+      let _scoreLastSentAt: number | null = null;
+      let _lastScore: number;
+      let _lastPlayerName: string;
+      let _lastErrorCode: number;
+
+      export const setPlayerScore = function (
+        leaderboardId: string,
+        score: gdjs.Variable,
```

Comment:
(The score and player name should be passed as `string`, though double check the parameters defined in the extension declaration)

Reply:
Alright !
The score too ? Even though it should be sent as a number to our API ?

Reply:
Ah you're right, a `float` is fine for the score (`float` is a type alias for `number` in TypeScript). So in the parameters of the extension, this will be an "expression" (which should have been just named "number", but that's how it's called).